text/chapter6.md

```diff
@@ -648,7 +648,7 @@ Another reason to define a superclass relationship is in the case where there is
      ```haskell
      instance actionMultiplyInt :: Action Multiply Int
      ```
-     Verify that this instance satisfies the laws listed above.
+     _Note_: Your instance must satisfy the type class laws listed above in order to pass the tests.
```

Comment:
Perhaps, just "Remember, your instance must satisfy the laws listed above." will do. Doesn't need to be wrapped in a _Note_. Passing tests should go without saying.

I agree "Verify" does sound like an extra step.